ci/test_lightning.py

```diff
@@ -0,0 +1,24 @@
+from subprocess import call
+from pathlib import Path
+
+from .build import build_tests
+
+source_files = [
+    'integrations/pytorch-lightning/Neptune-PyTorch-Lightning-basic.ipynb',
+    'integrations/pytorch-lightning/Neptune-PyTorch-Lightning-advanced.ipynb'
+]
+
+for filename in source_files:
+    build_tests(Path(filename))
+
+test_files = [
```

Comment:
@PiotrJander so as a contributor to this repo I should create a test_SOMETHING.py file and add all the "artifact" files I'd like to test in here right?